modules/installation-azure-marketplace-manifests.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_azure/installing-azure-customizations.adoc
+
+:_content-type: PROCEDURE
+[id="installation-azure-marketplace-manifests_{context}"]
+= Updating Manifests for Marketplace Installation
```

Comment:
```suggestion
= Updating manifests for marketplace installation
```
See the ISG: https://www.ibm.com/docs/en/ibm-style?topic=grammar-capitalization#headings-titles-and-banners

Reply:
Good point -  I think "Marketplace" should still be capitalized because it's a product name/official name of something.

Reply:
@bscott-rh I see further down where you always capitalize 'Marketplace', so ignore my capitalization of that word in the header.